src/Components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+function CreditCard(props) {
+  const {
+    type,
+    number,
+    expirationMonth,
+    expirationYear,
+    bank,
+    owner,
+    bgColor,
+    color,
+  } = props;
+
+  return (
+    <div
+      style={{
+        backgroundColor: `${bgColor}`,
+        color: `${color}`,
+      }}
+    >
+      <p>{type}</p>
+      <p>{number}</p>
```

Comment:
On the credit card you should render "**" followed by the last 4 numbers of the card